### PR TITLE
Eggcorns

### PR DESCRIPTION
**Pull request by kswedberg**
Added a list of "eggcorns": commonly mistaken English phrases most likely caused by hearing them rather than reading them.

### CODE REVIEW

**Comment by dariusk** [comment]
Woo, thanks!!

**Comment by Dewb** [comment]
Potential source for more: http://eggcorns.lascribe.net/browse-eggcorns/

File: data/words/eggcorns.json [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+      "mistakes": ["butt naked"],
+      "correct": "buck naked"
+    },
+    {
+      "mistakes": ["nip it in the butt"],
+      "correct": "nip it in the bud"
+    },
+    {
+      "mistakes": ["test your metal", "test your medal"],
+      "correct": "test your metal"
```
Comment (kejadlen) [inline data/words/eggcorns.json]: Shouldn't this be "test your mettle"?

File: data/words/eggcorns.json [hunk 2]
```diff
@@ -0,0 +1,179 @@
...
+      "mistakes": ["you've got another thing coming"],
+      "correct": "you've got another think coming"
+    },
+    {
+      "mistakes": ["statue of limitations"],
+      "correct": "statute of limtations"
+    },
+    {
+      "mistakes": ["ripe with conflict"],
+      "correct": ["rife with conflict"]
```
Comment (JeffreyATW) [inline data/words/eggcorns.json]: Should be a string?

**Comment by ajpiano** [comment]
Probably should also add "eggcorn" for "acorn" eh ??

File: data/words/eggcorns.json (hunk 1 above)
Comment (kswedberg) [inline data/words/eggcorns.json]: Yes! My bad. I'll update.

**Pull request merged**